src/sentry/static/sentry/app/views/performance/realUserMonitoring/transactionVitals.tsx

```diff
@@ -53,21 +53,31 @@ class TransactionVitals extends React.Component<Props> {
                 {results => (
                   <React.Fragment>
                     {vitals.map((vital, index) => {
+                      const details = WEB_VITAL_DETAILS[vital];
                       const error =
                         summaryResults.error !== null || results.error !== null;
-                      const alias = getAggregateAlias(
+                      const percentile_alias = getAggregateAlias(
                         `percentile(${vital}, ${PERCENTILE})`
                       );
-                      const summary =
-                        summaryResults.tableData?.data?.[0]?.[alias] ?? null;
+                      const count_alias = getAggregateAlias(`count_geq(${vital}, 0)`);
+                      const failed_alias = getAggregateAlias(
+                        `count_geq(${vital}, ${details.failureThreshold})`
+                      );
+                      const data = summaryResults.tableData?.data?.[0];
+                      const summary = (data?.[percentile_alias] ?? null) as number | null;
+                      const numerator = (data?.[failed_alias] ?? 0) as number;
+                      const denominator = (data?.[count_alias] ?? 0) as number;
+                      const failureRate =
+                        denominator <= 0 ? null : numerator / denominator;
```

Comment:
If you coalesced to 0 here you would have a simpler type in `VitalCard` and not need to handle nulls there.